### DISCUSSION TITLE
peerjs is not recognised as internal or external command

### DISCUSSION THREAD

**Discussion by kentctct** [peers/peerjs#1209] [Q&A]
hi after I run this line 
'npm install peer' in my project folder terminal
peer is installed just fine 
but when I start a peer server 'peerjs - - port 3000' 
the terminal returns 

peerjs is not recognised as internal or external command, operable program or batch file

I also tried 'npm install - g peer' 
peer is installed just fine 
but when I start the server the error is still the same

can anyone help? any help will be appreciated

**Comment by tamjeedhurcp** [comment]
I was having same issue with 
' npm install peer ' 
i figured it out by using "sudo npm install -g peer'
